dump/json_test.go

```diff
@@ -0,0 +1,52 @@
+package dump
+
+import (
+	"bytes"
+	"io/ioutil"
+	"path/filepath"
+	"regexp"
+	"testing"
+
+	"github.com/relex/fluentlib/testdata"
+	"github.com/stretchr/testify/assert"
+)
+
+var extPattern = regexp.MustCompile(`.ff$`)
```

Comment:
This pattern is declared twice, how move that to utils lib or anything like that?